openedx/core/djangoapps/schedules/models.py

```diff
@@ -23,3 +28,11 @@ class Schedule(TimeStampedModel):
     class Meta(object):
         verbose_name = _('Schedule')
         verbose_name_plural = _('Schedules')
+
+
+class ScheduleConfig(ConfigurationModel):
+    KEY_FIELDS = ('site',)
+
+    site = models.ForeignKey(Site)
```

Comment:
I believe we need to add a field for `create_schedules` in this model.  Otherwise, we can't do a course-specific staged rollout after enabling this global flag, per our [rollout plan](https://openedx.atlassian.net/wiki/spaces/RET/pages/162665873/Milestone+1+Automated+Communications+Engine+A.C.E.).   See step #3 and #6.